### DISCUSSION TITLE
Some random improvements

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/subfinder#473] [Ideas]
Great tool btw ❤❤

- It would be great to introduce a time to live for apikeys to avoid burning up all the queries.
- A template engine where anyone can add a source to his liking.

**Comment by olearycrew** [comment]
@ehsandeep thoughts on these? Should we open GitHub issues for these?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
